nservicebus/messaging/send-a-message.md

```diff
@@ -6,12 +6,17 @@ redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-To send a message, use the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver:
+Prior to V6 sending a message involved using the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver. In V6 the equivalent of the `IBus` is the `IBusContext` family of interfaces.
```

Comment:
I somehow don't like that we start of with version difference. I don't think the renaming of interfaces changes much in the scope of sending a basic message. So could we put the interface name and version a little bit in the background? Something like "[...] using the `Send` method on the `IBus`(pre v6)/`IBusContext`(v6) interface[...]". Maybe skip the version altogether an let the samples speak for themselves?